wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -28,6 +28,8 @@
   public static final String LINK_REL_ENVIRONMENT_OUTPUT_CHANNEL = "environment/outputChannel";
   public static final String LINK_REL_ENVIRONMENT_STATUS_CHANNEL = "environment/statusChannel";
 
+  public static final String WORKSPACE_START_DEBUG = "debug";
```

Comment:
can we have a more self-explaining name like `watch_container_logs_on_workspace_startup`?

Reply:
fixed